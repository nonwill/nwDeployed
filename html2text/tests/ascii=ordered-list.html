<html>
	<body>
		<ol>
			<li>this is thing one</li>
			<li>this is thing two</li>
		</ol>
	</body>
</html>
