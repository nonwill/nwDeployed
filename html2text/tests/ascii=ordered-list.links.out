ascii=ordered-list.default.out